core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/ClusterServiceRunner.java

```diff
@@ -51,13 +51,13 @@ public class ClusterServiceRunner {
     @Inject
     private GatewayService gatewayService;
 
-    public void runClusterManagerServices(StackDto stackDto) {
+    public void runClusterManagerServices(StackDto stackDto, boolean runPreServiceDeploymentForRecipe) {
```

Comment:
please rename to runPreServiceDeploymentRecipe everywhere

Reply:
why?

Reply:
the way you ask me to rename it makes it mean that its the recipe. please give me a valid reason

Reply:
I agree with @sodre90, the name of the variable should indicate the weather we should run those specific, pre-service-deployment, recipes or not.

Reply:
It controls if the PRE_SERVICE_DEPLOYMENT type recipes should run or not. I don't understand what is the "for" in pre service deployment for recipe?

Reply:
ok - i have made the changes. please review again